dts/arm/st/f4/stm32f437.dtsi

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2019, Markus Fuchs <markus.fuchs@de.sauter-bc.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/f4/stm32f407.dtsi>
```

Comment:
STM32F437 indeed inherits from STM32F407 but supports as well (among the IPs currently supported in zephyr):
-UART7 and 8
-SPI4, 5 and SPI6
-GPIOJ and K
Please add them here.

Reply:
Thanks. I have added these. I used stm32f429.dtsi as a template. As a side note, is it correct that stm32f429.dtsi defines i2s4, i2s5 and i2s6? From the documentation, it looks like there is only i2s2 and i2s3.

Reply:
IIRC, i2s4 5 and 6 are supported on all stm32f429 variants except the stm32f429**v**X ones.
So we delete them in stm32f429vX.dtsi file.

Reply:
Looking more into it, seems you're right about I2S instances. I'll fix that .